lib/src/loader.rs

```diff
@@ -218,9 +218,9 @@ fn load_elf(bytes: &[u8], name: String) -> Result<(Project,Machine)> {
 
     // for now we comment adding symbols from strippable symbol table:
     // we don't have an easy way/API to check if duplicate symbol/function targest have been added
-    // for sym in &binary.syms {
-    //     add_sym(&mut prog, sym, &binary.strtab);
-    // }
+     for sym in &binary.syms {
+         add_sym(&mut prog, sym, &binary.strtab);
+     }
```

Comment:
Does the function table dedup automatically now ? I.e., what changed such that the above comment doesn't apply?